phpstan.neon.dist

```diff
@@ -77,3 +77,9 @@ parameters:
         -
             message: '~^Access to undefined constant PDO::PGSQL_ATTR_DISABLE_PREPARES\.$~'
             path: %currentWorkingDirectory%/lib/Doctrine/DBAL/Driver/PDOPgSql/Driver.php
+
+        # False Positive
+        - '~Strict comparison using === between 1 and 2 will always evaluate to false~'
```

Comment:
@beberlei how is this a false positive?

https://github.com/doctrine/dbal/blob/85a983c3f8d6447c4441832477def1c28d57bee9/lib/Doctrine/DBAL/Internal/DependencyOrderCalculator.php#L106-L123

It looks like a bug in the code. How is `$vertex->state` expected to change its state from `IN_PROGRESS` to anything else between the first and last lines of the block above?